server/routers/trainers/trainers_api.py

```diff
@@ -0,0 +1,48 @@
+from unicodedata import name
+from fastapi import APIRouter
+from fastapi import Request, Response
+from . import trainers_utils
+from pymysql.err import IntegrityError
+router = APIRouter()
+
+
+@router.get('/trainer/{trainer_name}/pokemons', status_code=200)
+async def get_pokemon_of_trainers(trainer_name):
```

Comment:
you should check if the trainer name is valid, if not return HTTP_400_BAD_REQUEST